setup.py

```diff
@@ -0,0 +1,21 @@
+import os
+from git import Repo
+import sys
+path = os.path.abspath(os.path.dirname(__file__))
+PATH_TO_MODULE=os.path.join(path,'wiki_files')
+sys.path.append(PATH_TO_MODULE)
+if(os.path.exists(PATH_TO_MODULE)):
+    print('You have pywikibot repo downloaded')
+
+else:
+    print('Downloading pywikibot github repository')
+    Repo.clone_from('https://gerrit.wikimedia.org/r/pywikibot/core.git', 'wiki_files')
+    print("Downloaded pywikibot code from Github into the folder named wiki files")
+
+    link=input('Write the link for the wikipage : ')
```

Comment:
Leave space before and after the assignment operator

Reply:
why is that required

Reply:
It is the convention used for better code readability. Look up pep8speaks.